css/style.css

```diff
@@ -540,3 +540,143 @@ img{
     }
 }
 /***** End features *****/
+/***** contact *****/
+.contact{
+    background: linear-gradient(to right bottom, #7ed56f, #28b485);
+    padding: 15rem 0;
+}
+.contact form{
+    display: flex;
+    flex-direction: column;
+}
+.contact .title{
+    margin-bottom: 3rem;
+}
+#fullName, #emailAddress{
+    border: none;
+    padding: 0.75rem 0.875rem;
+}
+#fullName{
+    margin-bottom: 2rem;
+}
+.contactForm{
+    background: linear-gradient(to right, rgba(255,255,255,0.9) 0%, rgba(255,255,255,0.9) 100%),url(../assets/img/nat-10.jpg);
+    background-size: cover;
+    padding: 3rem;
+    width: 100%;
+}
+.contact .primaryButton{
+    font-size: 0.875rem;
+    padding: 0.75rem 2rem;
+    margin-top: 1.5rem;
+}
+input[type=radio]{
+    margin-right: 0.625rem;
+    appearance: none;
+    width: 2rem;
+    height: 2rem;
+    border: 4px solid var(--dark-green);
+    transition: .2 all linear;
+    border-radius: 50%;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    -o-border-radius: 50%;
+    -webkit-transition: .2 all linear;
+    -moz-transition: .2 all linear;
+    -ms-transition: .2 all linear;
+    -o-transition: .2 all linear;
+}
+input[type=radio]:checked{
+    border: 12px solid var(--dark-green);
+}
+.radioGroup {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-top: 1.25rem;
+}
+@media (min-width:1024px) {
+    .contactForm{
+        width: 100%;
+    background-image: linear-gradient(105deg, rgba(255,255,255,0.9) 0%, rgba(255,255,255,0.9) 50%, transparent 50%),url(../assets/img/nat-10.jpg);
+    background-size: 100%;
+    border-radius: 3px;
+    -webkit-box-shadow: 0 1.5rem 4rem rgb(0 0 0 / 20%);
+    box-shadow: 0 1.5rem 4rem rgb(0 0 0 / 20%);
+    }
+    .contact .primaryButton{
+        width: 11rem;
+    }
+    .contact form{
+        width: 41%;
+    }
+    .radioButtons{
+        display: flex;
+        margin-top: 2rem;
+    }
+    .radioGroup{
+        margin-top: 0;
+        margin-right: 0.625rem;
+        display: flex;
+        align-items: center;
+    }
+}
+/***** End contact *****/
+/***** footer *****/
+footer{
+    background-color: var(--footer-bg);
+    color: var(--footer-text);
+    padding: 4rem 0;
+}
+footer figure{
+    margin: 3rem 0;
+    width: 8rem;
+}
+.footerInfo{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: baseline;
+}
+.footerLinks{
```

Comment:
I recomend you to add something like `padding: 20px 0;` because now it looks like this. Background of the button which appears on hover overlays the line. If it's ok from UI/UX point of view for us, then it's fine.
<img width="413" alt="Screenshot 2022-03-18 at 20 32 45" src="https://user-images.githubusercontent.com/17166011/159063834-35a9e33f-2746-43f1-b3f6-9b8f849d5f57.png">